analyzetestgroups

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
```

Comment:
make the file executable with `chmod +x` as well, and possibly rename to end with `.py`.

also, can you add a short description in this file about what it is (e.g. copy the example output) and a line to the README?

consider trying out `black` on this file for consistent formatting

Reply:
“short description“: the ArgumentParser object already does that. Are you talking about a source code comment? (I’m happy to do this.)

Reply:
I was thinking a source code comment yes, but perhaps README + the argument parser description is good enough.

Reply:
Done in 7d67508